pkg/gline/app.go

```diff
@@ -158,7 +160,35 @@ func (m appModel) Update(msg tea.Msg) (tea.Model, tea.Cmd) {
 			return m, tea.Sequence(terminate, tea.Quit)
 
 		case "ctrl+c":
+			// Handle Ctrl-C: show abandoned command with ^C and move to new line
+			currentInput := m.textInput.Value()
+			if currentInput != "" {
+				// Print the current input with "^C" appended, then move to next line
+				fmt.Printf("%s^C\n", currentInput)
```

Comment:
seems we don't need the "if" above. this statement will do what we need for both cases, correct?

Reply:
Good catch, fixed.